drivers/media/v4l2-core/v4l2-ioctl.c

```diff
@@ -1333,6 +1333,7 @@ static void v4l_fill_fmtdesc(struct v4l2_fmtdesc *fmt)
 	case V4L2_META_FMT_UVC:		descr = "UVC Payload Header Metadata"; break;
 	case V4L2_META_FMT_D4XX:	descr = "Intel D4xx UVC Metadata"; break;
 	case V4L2_META_FMT_SENSOR_DATA:	descr = "Sensor Ancillary Metadata"; break;
+	case V4L2_META_FMT_BCM2835_ISP_STATS: descr = "BCM2835 ISP Image Statistics"; break;
```

Comment:
Do we need to make it _BCM283X_ as it applies to 2835/2836/2837, and of course the anomalous 2711?

We do have the inconsistency there in that the drivers are bcm2835-camera, bcm2835-codec, bcm2835-unicam, but I can see the user comments otherwise.

Reply:
Good point.  I suppose you could argue that the HW in 2836/2837/2711 has not changed so the BCM2835 name is still "right".

If we want to be consistent with the other drivers, perhaps we keep it as-is?